packages/client/tests/functional/issues/13097-group-by-enum/tests.ts

```diff
@@ -64,5 +64,10 @@ testMatrix.setupTestSuite(
       from: ['sqlite', 'sqlserver'],
       reason: "SQLite and SQLServer don't support enums",
     },
+    skipProviderFlavor: {
+      from: ['js_neon', 'js_pg'],
+      reason:
+        "Inconsistent column data: List field did not return an Array from database. Type identifier was Enum(EnumId(2)). Value was Enum(Some('{A}')).",
```

Comment:
Looks like something we need to investigate later

Reply:
This error is coming from a a ConversionError in our `sql_query_connector` crate. It looks like that, when using Driver Adapters, we're either writing wrong data to the database, or we're marking their types in a faulty way that affects Quaint.